README.md

```diff
@@ -1 +1,168 @@
-# ecdsa-xi-2023-cryptosuite
+# ECDSA XI 2023 Data Integrity Cryptosuite _(@digitalbazaar/ecdsa-xi-2023-cryptosuite)_
+
+[![Build status](https://img.shields.io/github/actions/workflow/status/digitalbazaar/ecdsa-xi-2023-cryptosuite/main.yml)](https://github.com/digitalbazaar/ecdsa-xi-2023-cryptosuite/actions?query=workflow%3A%22Node.js+CI%22)
+[![Coverage status](https://img.shields.io/codecov/c/github/digitalbazaar/ecdsa-xi-2023-cryptosuite)](https://codecov.io/gh/digitalbazaar/ecdsa-xi-2023-cryptosuite)
+[![NPM Version](https://img.shields.io/npm/v/@digitalbazaar/ecdsa-xi-2023-cryptosuite.svg)](https://npm.im/@digitalbazaar/ecdsa-xi-2023-cryptosuite)
+
+> ECDSA XI 2023 Data Integrity Cryptosuite for use with jsonld-signatures.
+
+## Table of Contents
+
+- [Background](#background)
+- [Security](#security)
+- [Install](#install)
+- [Usage](#usage)
+- [Contribute](#contribute)
+- [Commercial Support](#commercial-support)
+- [License](#license)
+
+## Background
+
+For use with https://github.com/digitalbazaar/jsonld-signatures v11.0 and above.
+
+See also related specs:
+
+* [Verifiable Credential Data Integrity](https://w3c.github.io/vc-data-integrity/)
+
+## Security
+
+TBD
+
+## Install
+
+- Browsers and Node.js 18+ are supported.
+
+To install from NPM:
+
+```
+npm install @digitalbazaar/ecdsa-xi-2023-cryptosuite
+```
+
+To install locally (for development):
+
+```
+git clone https://github.com/digitalbazaar/ecdsa-xi-2023-cryptosuite.git
+cd ecdsa-xi-2023-cryptosuite
+npm install
+```
+
+## Usage
+
+The following code snippet provides a complete example of digitally signing
+a verifiable credential using this library:
+
+```javascript
+import * as EcdsaMultikey from '@digitalbazaar/ecdsa-multikey';
+import {DataIntegrityProof} from '@digitalbazaar/data-integrity';
+import {cryptosuite as ecdsaRdfc2019Cryptosuite} from
```

Comment:
Could this example be fixed up?  Copy & paste issues like this remain.  Also I don't think this example even uses the XI part of the lib?  At a minimum, point at `testSignVerify.js` as a (hopefully) functioning example of how to use the library.